packages/shared/src/types/utilities/Pretty.ts

```diff
@@ -0,0 +1,3 @@
+export type Pretty<K> = {
```

Comment:
Can you explain to me what the benefit of this type is?

Reply:
It flattens the object so the type that you see in the editor is human-friendly instead of a nasty union of several types 😬